### PR TITLE
feat(bottom sheet): INFORMATIVE and ACTIONS BottomSheet

### PR DESCRIPTION
**Pull request by atabel**
https://jira.tid.es/browse/WEB-673

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -62,5 +62,14 @@ export {renewSession, onSessionRenewed, logout} from './src/sessions';
 
 export {showAppRating} from './src/app-rating';
 
-export {bottomSheet, bottomSheetSingleSelector} from './src/bottom-sheet';
-export type {SheetListItem} from './src/bottom-sheet';
+export {
```
Comment (pladaria) [inline index.ts:65]: doc update?

File: index.ts [hunk 2]
```diff
@@ -62,5 +62,14 @@ export {renewSession, onSessionRenewed, logout} from './src/sessions';
...
+    bottomSheet,
+    bottomSheetSingleSelector,
+    bottomSheetActionSelector,
+    bottomSheetInfo,
```
Comment (pladaria) [inline index.ts:69]: Perhaps the bridge responsibility should be to provide the bottomSheet feature.

The bottomSheetSingleSelector, bottomSheetActionSelector, bottomSheetInfo or any other future variant should be implemented in the application.

Comment (atabel) [reply index.ts:69]: I think these wrapper functions are quite convenient for the final apps (webapp, solar) to use, given the SheetUI model is quite complex. I think this is the best place to have these functions. In mistica we'll have the web implementations of these components, not these helper functions.

File: index.ts (hunk 1 above)
Comment (atabel) [reply index.ts:65]: sure

Comment (atabel) [reply index.ts:65]: done

**Review by brtbrt — APPROVED** [review]

**Pull request merged**